lib/mixpanel-node.js

```diff
@@ -143,16 +148,16 @@ var create_client = function(token, config) {
             }
             try {
                 body = JSON.parse(body);
+                if (body.status === 0) {
```

Comment:
If we are preparing the worst, then `null` isn't the only possibility. If the "verbose" flag is missing (or, for some reason, not parsed by the server), then the Mixpanel API can also return "0" or "1" as the body of the response. By moving the `if` there, I just don't care about the exact cause.